iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Area.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+/**
+ * @author Marine Guibert {@literal <marine.guibert at artelys.com>}
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+public interface Area extends Identifiable<Area> {
+
+    AreaType getAreaType();
+
+    void setAreaType(AreaType areaType);
```

Comment:
This method should not be available in the interface.
It was added only to manage the merge case, but having it in the interface may indicate that this is a "regular" operation (plus, in the actual implementation, nothing unsure that no VoltageLevel of the current Area is not already linked to another Area of the new AreaType).

Reply:
I agree, I changed the merge implementation so that we don't need this anymore